### PR TITLE
feat: add PBF_PATH env. variable for alternative data import

### PR DESCRIPTION
**Pull request by Bartizan**
See at #247

### CODE REVIEW

**Comment by leonardehrenfried** [comment]
To me this looks like a good PR, however I would ask for the following improvements:

- Can you throw an error when both URL and path are defined? A casual user could reasonably expect that both would be imported but this is not the case, isn't it? Can you also make this clear in the docs?
- Could you add an integration test for this feature? You'd probably have to disable it for 3.6 since that doesn't have that feature - use a variable in the matrix build definition.

File: 3.7/README.md [hunk 1]
```diff
@@ -94,6 +95,26 @@ docker run -it --rm --shm-size=1g \
...
+A sample of the docker configuration with PBF_PATH variable (point at the docker image that supports PBF_PATH) is:
+
+``` sh
+docker run -it --rm \
+  -e PBF_PATH=/nominatim/data/monaco-latest.osm.pbf \
+  -e REPLICATION_URL=https://download.geofabrik.de/europe/monaco-updates/ \
+  -p 8080:8080 \
+  -v C:\osm-maps\data:/nominatim/data \
+  --name nominatim \
+  mediagis/nominatim:3.7-pbf_path
```
Comment (leonardehrenfried) [inline 3.7/README.md:115]: I think you need change this before we merge.

Comment (Bartizan) [reply 3.7/README.md]: fixed

**Comment by Bartizan** [comment]
be back with suggested improvements

**Comment by leonardehrenfried** [comment]
Now that I think about it, the integration test doesn't have the highest priority should you struggle with it.

**Review by leonardehrenfried — APPROVED** [review]

**Comment by leonardehrenfried** [comment]
I think this is a useful addition and that this PR is ready for merging. However, I will let @philipkozeny do the final review.

**Review by philipkozeny — APPROVED** [review]
LGTM, thanks for the addition!

**Pull request merged**